src/topology/metric_space/basic.lean

```diff
@@ -402,7 +402,7 @@ def ball (x : α) (ε : ℝ) : set α := {y | dist y x < ε}
 
 @[simp] theorem mem_ball : y ∈ ball x ε ↔ dist y x < ε := iff.rfl
 
-theorem mem_ball' : y ∈ ball x ε ↔ dist x y < ε := by rw dist_comm; refl
+theorem mem_ball' : y ∈ ball x ε ↔ dist x y < ε := by rw [dist_comm, mem_ball]
```

Comment:
Another way to spell this would be
```lean
theorem mem_ball' : y ∈ ball x ε ↔ dist x y < ε := dist_comm y x ▸ iff.rfl
```
but that's not really an improvement